L1Trigger/TrackFindingTracklet/src/TrackletProcessor.cc

```diff
@@ -453,20 +455,16 @@ void TrackletProcessor::execute() {
               if (rbin - rbinfirst > rdiffmax)
                 continue;
 
-              unsigned int irouterbin = vmbits >> 2;
+              int rzbin = outervmstub.vmbits().bits(0, 3);
 
               FPGAWord iphiinnerbin = innervmstub.finephi();
               FPGAWord iphiouterbin = outervmstub.finephi();
 
-              unsigned int index = (irouterbin << (outerphibits_ + innerphibits_)) +
-                                   (iphiinnerbin.value() << outerphibits_) + iphiouterbin.value();
+              unsigned int index = (iphiinnerbin.value() << outerphibits_) + iphiouterbin.value();
 
-              assert(index < phitable_[phiindex].size());
-              if (!phitable_[phiindex][index]) {
-                if (settings_.debugTracklet()) {
-                  edm::LogVerbatim("Tracklet") << "Stub pair rejected because of tracklet pt cut";
-                }
-                continue;
+              if (iSeed_ >= 4) {  //Also use r-position
```

Comment:
magic numbers should be named constants (4, 3, 1, 2, another 3...)

Reply:
made named